sys/sys/syscallsubr.h

```diff
@@ -385,6 +386,15 @@ int	kern_writev(struct thread *td, int fd, struct uio *auio);
 int	kern_socketpair(struct thread *td, int domain, int type, int protocol,
 	    int *rsv);
 int	kern_unmount(struct thread *td, const char *path, int flags);
+int	kern_kmq_notify(struct thread *, int, struct sigevent *);
```

Comment:
Please put functions by alphabetical order in the file

Reply:
> Please put functions by alphabetical order in the file

Fixed.